### PR TITLE
Fix issue:80, First item is not shown when hint is used

### PR DESCRIPTION
**Pull request by Hamdy-khader**
Issue:
https://github.com/jaredrummler/MaterialSpinner/issues/80

### CODE REVIEW

File: library/src/main/java/com/jaredrummler/materialspinner/MaterialSpinnerAdapter.java [hunk 1]
```diff
@@ -30,17 +30,26 @@ public MaterialSpinnerAdapter(Context context, List<T> items) {
   }
 
   @Override public int getCount() {
-    int size = items.size();
-    if (size == 1) return size;
-    return size - 1;
+    if (isHintEnabled()){
+      return items.size();
+    }else{
```
Comment (jaredrummler) [inline library/src/main/java/com/jaredrummler/materialspinner/MaterialSpinnerAdapter.java]: Please format

File: library/src/main/java/com/jaredrummler/materialspinner/MaterialSpinnerAdapter.java [hunk 2]
```diff
@@ -30,17 +30,26 @@ public MaterialSpinnerAdapter(Context context, List<T> items) {
...
+    }
   }
 
   @Override public T getItem(int position) {
-    if (position >= getSelectedIndex() && items.size() != 1) {
+    if (isHintEnabled()){
+      return items.get(position);
+
+    }else{
+      if (position >= getSelectedIndex() && items.size() != 1) {
```
Comment (jaredrummler) [inline library/src/main/java/com/jaredrummler/materialspinner/MaterialSpinnerAdapter.java]: move inner if statement to an else if

File: library/src/main/java/com/jaredrummler/materialspinner/MaterialSpinnerAdapter.java (hunk 1 above)
Comment (Hamdy-khader) [reply library/src/main/java/com/jaredrummler/materialspinner/MaterialSpinnerAdapter.java]: done

File: library/src/main/java/com/jaredrummler/materialspinner/MaterialSpinnerAdapter.java (hunk 2 above)
Comment (Hamdy-khader) [reply library/src/main/java/com/jaredrummler/materialspinner/MaterialSpinnerAdapter.java]: done

**Comment by jaredrummler** [comment]
Thank you

**Comment by Hamdy-khader** [comment]
NP

**Pull request merged**